modules/monitoring/manifests/checks.pp

```diff
@@ -58,6 +59,36 @@
 
   include icinga::plugin::check_http_timeout_noncrit
 
+  $content_data_api_hostname    = "content-data-api.${app_domain}"
+  $content_data_api_metrics_url = '/healthcheck/metrics'
+  $content_data_api_search_url = '/healthcheck/search'
```

Comment:
Is it worth renaming these from `_url` to `_path`?

Reply:
Yes, that seems like a good idea 👍